content/docs/add-custom-view.md

```diff
@@ -12,80 +12,47 @@ categories:
 image: "/images/step-extension.png"
 ---
 
-Kaoto's frontend is extendable, allowing you to create custom views that you
-can configure to show for specific steps. We call these custom views **Step
-Extensions**.
+Kaoto's frontend is extendable, allowing you to create custom views. You
+can configure Kaoto to show a specific view instead of the generic form to configure properties in a step. 
+We call these custom views **Step Extensions**.
 
-![Example of a Step Extension](/images/step-extension.png)
+Kaoto has an **Step Extension API** to interact with the main application. This API can
+be used to control deployments remotely, or to do some kind of data transformation 
+with the output of a particular step.
 
-You can use the Step Extension API to interact with Kaoto's internal state,
-whether that be to control deployments remotely, or to do some kind of data
-transformation with the output of a particular step.
+There are two main changes when adding a new step extension:
```

Comment:
Why "changes"? It could be a completely new application

Reply:
"parts" didn't sound right to me, but you are right, "changes" is not the word either.